build_projects/shared-build-targets-utils/project.json

```diff
@@ -6,6 +6,7 @@
     "Microsoft.CSharp": "4.0.1",
     "System.Dynamic.Runtime": "4.0.11",
     "System.Reflection.Metadata": "1.3.0",
+    "System.Runtime.Extensions":"4.1.0",
```

Comment:
That is in NETStandard.Library. https://github.com/dotnet/corefx/blob/master/pkg/NETStandard.Library/NETStandard.Library.packages.targets#L30 

What issue were you hitting?